src/main/java/oscar/form/study/HSFO/pageUtil/HsfoQuartzServlet.java

```diff
@@ -72,19 +69,23 @@ public static void schedule() throws Exception {
 
         scheduler.start();
 
```

Comment:
lots done here - I assume with respect to quartz compatibility in new version.  need to explain that in a commit.  ideally its own for it, but it should be stated more specifically what's going on in this commit rather than general dependency statement

Reply:
Added commit description about Quartz imports.